tendrl/common/tests/test_config.py

```diff
@@ -8,11 +8,9 @@ def setup_method(self, method):
         config.os.path.exists = MagicMock(return_value=True)
 
     def test_Tendrlconfig(self, monkeypatch):
-        config.TendrlConfig()
+        config.TendrlConfig("module", "/etc/tendrl/tendrl.conf")
```

Comment:
tests should not assume local files existence, please create a temp conf and supply that, dont hardcode